soccer/planning/trajectory/Trajectory.hpp

```diff
@@ -0,0 +1,130 @@
+#pragma once
+
+#include <memory>
+#include <planning/MotionInstant.hpp>
+#include <planning/paths/InterpolatedPath.hpp>
+#include <planning/paths/Path.hpp>
+#include <utility>
+
+namespace Trajectory {
+using Planning::AngleInstant, Planning::MotionInstant, Planning::Path,
+    Planning::InterpolatedPath, Planning::RobotInstant;
+
+using AngleFn = std::function<AngleInstant(MotionInstant)>;
+
+/**
+ * \brief Represents a trajectory x(t), y(t), h(t) that is sent from the
+ * PlannerNode to the MotionControlNode.
+ *
+ * TEMP: For simplicity, this is just a angleFunction attached to a
+ * InterpolatedPath.
+ */
+class Trajectory {
+public:
+    /**
+     * \brief Default constructor so we can make an array of this.
+     */
+    Trajectory() = default;
```

Comment:
Should probably have default copy and move constructors and assignment operators.

Reply:
Aren't these implicitly defined by the compiler?

Reply:
Mb, only needed if you user-declare a destructor. This is fine.